build/commands/lib/config.js

```diff
@@ -87,6 +88,9 @@ const parseExtraInputs = (inputs, accumulator, callback) => {
 }
 
 const Config = function () {
+  // TODO(yannic): Should the user be able to override this?
```

Comment:
so? We either do it or not :) don't need to left TODOs.

Reply:
```suggestion
```